lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/ClassHierarchyDeserializationTest.java

```diff
@@ -35,16 +35,21 @@
 
 /**
  * Test case for class hierarchy deserialization.
+ * TODO: The files should be created and deserialized by the AvroClassHierarchySerializer (REEF-400)
  */
 public class ClassHierarchyDeserializationTest {
+  private final ConfigurationSerializer configurationSerializer = new AvroConfigurationSerializer();
+  private final ClassHierarchySerializer classHierarchySerializer = new AvroClassHierarchySerializer();
 
   /**
    * generate task.bin from running .Net ClassHierarchyBuilder.exe
    */
   @Test
   public void testDeserializationForTasks() {
+    // TODO: The file should be written by Avro (REEF-400)
```

Comment:
@jwang98052 In this case, I don't think it would be good idea to have a separate class as `TestClassHierarchyRoundTrip`. Once we make our Avro version default serializer, there is no need to test the implementation respectively. Instead, I replicate the methods which use the protobuf serializer. How does this look?